private-ai-services/paisconfiguration.yaml

```diff
@@ -0,0 +1,57 @@
+# Example PAISConfiguration
+#
+# Please review and edit where needed
+# Several dependencies, like ConfigMaps and Secrets, must be created too.
+---
+apiVersion: pais.vmware.com/v1alpha1
+kind: PAISConfiguration
+metadata:
+  name: default
+  namespace: my-namespace-on-supervisor
+spec:
+  worker:
+    storageClassName: my-preferred-storage-class
+  clientTls:
+    # These are optional
+    # Each should be a ConfigMap with a single "ca.crt" field
```

Comment:
Would it help to specify any formatting information about that certificate? I'd guess PEM-formatted cert in a multiline string, but various APIs across Supervisor have not been consistent about this (some base64-encode the PEM-formatted cert). It might not be obvious to consumers, but feel free to ignore if you have detailed API docs somewhere else.